testinfra/test_ami_nix.py

```
@@ -787,7 +787,9 @@ def test_postgrest_read_only_session_attrs(host):
             print(
                 f"\nFound 'session is not read-only' errors in PostgREST logs:\n{result['stdout']}"
             )
-            assert False, "PostgREST logs contain 'session is not read-only' errors even though PostgreSQL is configured for read-only mode"
+            assert False, (
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)